string droneName
string newGroupName
---
string result
